config/development.json

```diff
@@ -82,6 +82,7 @@
 		"jetpack/connect-redirect-pressable-credential-approval": true,
 		"jetpack/connect/remote-install": true,
 		"jetpack/connect/mobile-app-flow": true,
+		"jetpack/connect/woo-dna": true,
```

Comment:
Having a hard time finding this in the docs - which enviornment does this turn it on for? Only `calypso.localhost:3000`?

Reply:
Yes, only locally for now.

Reply:
What's the plan for turning the feature on for staging & production?

Reply:
We'll first merge this into `master` as-is, and wait a bit in case there are regressions. Before the "call for testing", we'll turn the feature on in `staging` so people can test it without running Calypso locally. After that we'll enable it for `production`, wait a couple of days more for regressions, and after that we'll be ready to release a version of the plugin that uses the new flow.

How does that sound? I just made it up, there was no real "plan" until now :)

Reply:
@DanReyLop I like the plan but I expect we need someone from calypso team to review and approve it.